jenkins/job_manager/test/SettingsTest.groovy

```diff
@@ -51,13 +62,17 @@ class SettingsTest {
                 modules:
                     - check_cache
                     - code_coverage'''
-        def configDefault = new Settings(simpleSettingsYaml, loggerMock)
+
+        def SettingsParserDefault = new SettingsParser(simpleSettingsYaml)
```

Comment:
variables should be camel case https://www.quora.com/What-is-the-difference-between-Pascal-Case-and-Camel-Case